var/spack/repos/builtin/packages/py-pytouchreader/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPytouchreader(PythonPackage):
+    """Python interface to interact with touch files."""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/PyModules"
+    url      = "ssh://bbpcode.epfl.ch/hpc/PyModules"
+
+    version('develop', git=url, clean=False)
+
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
I think setuptools is typically build dependency only

Reply:
Since it creates a script entry point it's a run-time dependency, too.

Otherwise, it'll fail with some `pkg_resources` error.